server/app/uk/gov/ons/addressIndex/server/modules/SystemBootstrap.scala

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.addressIndex.server.modules
+
+import java.io.File
+import javax.inject.{Inject, Singleton}
+import play.api.Logger
+
+@Singleton
+class SystemBootstrap @Inject()() {
+  System.load(
+    new File(
+      "parsers/src/main/resources/libbackend.so"
```

Comment:
This would be better in a config so that we will be able to change it without re-assembling the application.
Additionally consider that we will need to load specific file depending on which OS the application is currently running (maybe it's a future story).